packages/serverpod/lib/src/util/path_util.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:path/path.dart' as p;
+
+/// Provides utility functions for working with paths.
+class PathUtil {
+  /// Tests if a file is in a whitelisted path. File paths should all be
+  /// relative. Complete directories can be whitelisted by adding a trailing
+  /// slash. The path and whitelisted paths are case sensitive and should be
+  /// in unix/posix format.
+  static bool isFileWhitelisted(String path, Set<String> whitelistedFiles) {
+    for (var whitelistedFile in whitelistedFiles) {
```

Comment:
Suggestion: use `any` and return the result.

Seems like we are replicating the functionality of the `any` list method.

Reply:
That was much shorter! :)